### PR TITLE
Feat/ddm hookup

### PR DESCRIPTION
**Pull request by elijaharita**
not complete, but working in its current state

### CODE REVIEW

File: components/Button.module.scss [hunk 1]
```diff
@@ -26,4 +26,10 @@
   &:hover {
     box-shadow: 0 5px 15px rgba(0, 0, 0, 0.3);
   }
-}
+
+  &:disabled {
+    cursor: default;
```
Comment (jimmylee) [inline components/Button.module.scss:31]: I like using `cursor: not-allowed` for this case.

**Comment by jimmylee** [comment]
This is looking great Elijah :-)

File: components/Button.module.scss (hunk 1 above)
Comment (elijaharita) [reply components/Button.module.scss]: fixed this!

**Review by jimmylee — APPROVED** [review]
LGTM

**Pull request merged**